health.go

```diff
@@ -80,6 +81,7 @@ func NewDefaultHealthCheck() *HealthCheck {
 		GracePeriodSeconds:     30,
 		IntervalSeconds:        10,
 		TimeoutSeconds:         5,
+		DelaySeconds:           15,
```

Comment:
Thanks for the pointer.

Shouldn't Marathon pick the default if `delaySeconds` isn't passed along the API request?